app/helpers/model_files_helper.rb

```diff
@@ -4,8 +4,9 @@ def slicer_links(file)
     # i18n-tasks-use t('model_files.download.orca')
     # i18n-tasks-use t('model_files.download.prusa')
     # i18n-tasks-use t('model_files.download.bambu')
+    # i18n-tasks-use t('model_files.download.elegoo')
```

Comment:
If you run `i18n-tasks add-missing -l en` it should fill in the missing translation string for you in the right place.